src/Meplato.Store2.Tests/Availabilities/GetTests.cs

```diff
@@ -0,0 +1,59 @@
+﻿#region Copyright and terms of services
+
+// Copyright (c) 2015 Meplato GmbH.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License. You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License
+// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+// or implied. See the License for the specific language governing permissions and limitations under
+// the License.
+
+#endregion
+
+using System.Threading.Tasks;
+using NUnit.Framework;
+
+namespace Meplato.Store2.Tests.Availabilities
+{
+    [TestFixture]
+    public class GetTests : TestCase
+    {
+        [Test]
+        public async Task TestGet()
+        {
+            MockFromFile("availabilities.get.success");
+
+            var service = GetAvailabilitiesService();
+            Assert.NotNull(service);
+
+            var response = await service.Get().Spn("1234").Do();
+            Assert.NotNull(response);
+            Assert.NotNull(response.Kind);
+            Assert.AreEqual("store#availabilities/getResponse", response.Kind);
+            Assert.NotNull(response.Items);
+            foreach (var availability in response.Items)
+            {
+                Assert.AreEqual("1234", availability.Spn);
+            }
+        }
+
+        [Test]
+        public void TestGetNotFound()
+        {
+            MockFromFile("availabilities.get.not_found");
+
+            var service = GetAvailabilitiesService();
+            Assert.NotNull(service);
+
+            var response = await service.Get().Spn("no-such-product").Do();
+            Assert.NotNull(response);
+            Assert.NotNull(response.Kind);
+            Assert.AreEqual("store#availabilities/getResponse", response.Kind);
+            Assert.Null(response.Items);
```

Comment:
Test will fail, because an exception is thrown for an unknown product, so you need to change the assertion:

```suggestion
            var ex = Assert.ThrowsAsync<ServiceException>(() => service.Get().Spn("no-such-product").Do());
            Assert.NotNull(ex);
```